src/cli.ts

```diff
@@ -73,6 +73,21 @@ const handleJobExecution = (input: string, _: unknown) => {
   return input;
 };
 
+const handleOnboardingType = (input: string, _: unknown) => {
+  if (input === 'preconnect') {
+    return 'preconnect';
+  }
+
+  if (input === 'jitp') {
+    return input;
+  }
+
+  new Log(false).error(
+    'Input for associate must be blank, "jitp" or "preconnect"',
```

Comment:
I think 
~~Input for associate must be blank,~~
should become
`Input for onboarding type must be`